jenkins-build.sh

```diff
@@ -408,12 +463,14 @@ then
   $USE_SUDO sed -i 's/^\(\s*server\s\+s3-instance.* check\).*$/\1/g' /etc/haproxy/haproxy.cfg
 fi
 
-# Dump last log lines for easy lookup in jenkins
-tail -50 /var/log/seagate/s3/s3server.INFO
-
 # To debug if there are any errors
 tail -50 /var/log/seagate/s3/s3server.ERROR || echo "No Errors"
 
+if [ ! -z "$generate_support_bundle" ]
```

Comment:
Please add comment here e.g. jenkins pipeline will trigger this if we pass certain parameter

Reply:
sure.